Sources/OutputTarget.swift

```diff
@@ -32,7 +32,7 @@ import Darwin.C
 
 private func getEnvValue(key: String) -> String? {
     let value = getenv(key)
-    return value != nil ? String.fromCString(value) : nil
+    return value != nil ? String(cString: value!) : nil
```

Comment:
No problem at all!

Since we have already checked value is not `nil` it is safe to do an explicit unwrapping.

However, I would prefer to write it as below now:

```swift
private func getEnvValue(key: String) -> String? {
    guard let value = getenv(key) else {
        return nil
    }
    return String(cString: value)
}
```

I am not sure why I wrote that code XD...